wgpu/gui/jupyter.py

```diff
@@ -48,7 +67,8 @@ def handle_event(self, event):
             self._pixel_ratio = event["pixel_ratio"]
             self._logical_size = event["width"], event["height"]
 
-        super().handle_event(event)
+        ev = EVENT_TYPE_MAP(event_type)(event_type, **event)
```

Comment:
Interesting how little change is needed here :)

If we go down this path, we should probably update the jupyter_rfb spec to say that events may be dict-like.